Apps/WebClient/src/ClientApp/test/healthInsights.test.ts

```diff
@@ -0,0 +1,82 @@
+import { createLocalVue, shallowMount, Wrapper } from "@vue/test-utils";
+import VueContentPlaceholders from "vue-content-placeholders";
+import VueRouter from "vue-router";
+import Vuex, { Store } from "vuex";
+
+import { SERVICE_IDENTIFIER } from "@/plugins/inversify";
+import container from "@/plugins/inversify.config";
+import { ILogger } from "@/services/interfaces";
+import { GatewayStoreOptions, RootState } from "@/store/types";
+import HealthInsightsView from "@/views/healthInsights.vue";
+
+import { storeStub } from "./stubs/store/store";
+import { DateTime } from "luxon";
+
+let store: Store<RootState>;
+
+function createWrapper(
+    options?: GatewayStoreOptions
+): Wrapper<HealthInsightsView> {
+    const localVue = createLocalVue();
+    localVue.use(Vuex);
+    localVue.use(VueRouter);
+    localVue.use(VueContentPlaceholders);
+    if (options === undefined) {
+        options = storeStub;
+    }
+
+    store = new Vuex.Store(options);
+
+    return shallowMount(HealthInsightsView, {
+        localVue,
+        store: store,
+        stubs: {
+            "font-awesome-icon": true,
```

Comment:
This will need to be updated to hg-icon per 8a27fbefe1fe6d235f4d2268b7bca33aa9c83213.